src/masoniteorm/schema/Blueprint.py

```diff
@@ -428,7 +443,7 @@ def primary(self, column):
         Returns:
             self
         """
-        self.table.set_primary_key([column])
+        self.table.set_primary_key(column)
```

Comment:
@josephmancuso This was causing a bug so I fixed it, is it correct ?

Reply:
Uhh yeah looks right. I have to check the table tests